source/module_hamilt_lcao/module_dftu/dftu_hamilt.cpp

```diff
@@ -86,7 +86,7 @@ void DFTU::cal_eff_pot_mat_real(const int ik, double* eff_pot, const std::vector
             &GlobalV::NLOCAL, &GlobalV::NLOCAL, &GlobalV::NLOCAL,
             &half, 
             ModuleBase::GlobalFunc::VECTOR_TO_PTR(VU), &one_int, &one_int, this->LM->ParaV->desc, 
-            this->LM->Sloc.data(), &one_int, &one_int, this->LM->ParaV->desc,
+            sk, &one_int, &one_int, this->LM->ParaV->desc,
```

Comment:
delete this->LM?

Reply:
I have deleted all use of LM in module_dftu.